subsys/logging/Kconfig.processing

```diff
@@ -90,6 +90,17 @@ config LOG_PROCESS_THREAD_STACK_SIZE
 	help
 	  Set the internal stack size for log processing thread.
 
+config LOG_PROCESS_THREAD_SPECIFY_THREAD_PRIORITY
+	bool "Enable specifying the priority of the internal log processing thread"
```

Comment:
There was an action to remove prompts starting with "Enable ..." #43586. Maybe something like "Set custom thread priority". `internal` might not be needed as this is already dependent on option enabling internal thread.